public/assets/homepage.js

```diff
@@ -203,6 +203,13 @@ $( document ).ready(function() {
         logWebpageActivity("Click_module=StartExploring_location=Index");
     });
 
+    //Triggered when the city or mapathon links are clicked
+    // Logs"Click_module=ExploreMapLink_Location=<cityName or "mapathon">
+    $(".exploremaplink").on("click", function(e){
+        var cityName = e.currentTarget.id;
+        logWebpageActivity("Click_module=ExploreMapLink_Location="+cityName);
```

Comment:
Upon thinking a bit harder about this, I don't think that "ExploreMapLink" is a good name for what they are clicking on, and "Location" is used elsewhere in this logging to mean which part of the website it is being logged from. So I think something more like `"Click_module=OtherCityLink_City="+cityName` would be better.

Reply:
Which also makes me realize "exploremaplink" is not a great name for this class either! Would you mind renaming the class to something that better describes what the class is for?

Reply:
Oh also won't this log "Click_module=ExploreMapLink_Location=mapathon" when the mapathon link is clicked?

Reply:
Yeah, for right now it would log "Click_module=ExploreMapLink_Location=mapathon" when the mapathon link is clicked.

Reply:
That does not seem quite right on second thought, I think I could change the ID of the mapathon link to reflect the city name if that is better.

Reply:
I have just set it to log "Click_module=OtherCityLink_city=NewbergMapathon", is that an improvement?

Reply:
Nah, city doesn't matter for mapathon link. Just need to know it is a mapathon link.

Reply:
>I have just set it to log "Click_module=OtherCityLink_city=NewbergMapathon", is that an improvement?

No that's not great because it is not a link to another city. It is just a link to mapathon event. The links happen to be using the same CSS class, but they are not all referencing another city.

Reply:
Gotcha, it now logs "Click_module=OtherCityLink_City=mapathonLink"

Reply:
Class name is now class="otherCityLinks" as well.

Reply:
But `Click_module=OtherCityLink_City=mapathonLink` still has two references to there being other cities. All you need is something like `Click_module=MapathonLink`